epoch.go

```diff
@@ -1649,6 +1649,7 @@ func (e *Epoch) createBlockVerificationTask(block Block, from NodeID, vote Vote)
 		verifiedBlock, err := block.Verify(context.Background())
 		if err != nil {
 			e.Logger.Debug("Failed verifying block", zap.Error(err))
+			e.triggerEmptyBlockNotarization(md.Round, fmt.Sprintf("Invalid block: %s", err))
```

Comment:
nit: feel free to ignore, but I feel like its weird to add a reason param here. Especially since we log the error, then call triggerEmptyBlockNotarization and log the error again. Maybe we just add to the log above that we are triggering an empty block notarization?

```suggestion
			e.Logger.Debug("Failed verifying block, triggering empty block notarization", zap.Error(err))
			e.triggerEmptyBlockNotarization(md.Round)
```

Reply:
removed the parameter and moved the log to the caller functions